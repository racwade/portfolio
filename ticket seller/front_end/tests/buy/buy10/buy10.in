login
test_buypoor
buy
test_sell
Few
1
logout