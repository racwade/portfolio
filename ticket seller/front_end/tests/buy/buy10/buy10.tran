00 test_buypoor        BS 000000000