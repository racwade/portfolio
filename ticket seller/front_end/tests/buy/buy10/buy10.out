welcome to the ticket seller system
enter username:
logged in as test_buypoor
enter seller's username:
enter event name:
enter number of tickets to buy:
error: not enough credit to buy that many tickets
bye